alliance/config/settings/base.py

```diff
@@ -140,8 +160,9 @@ def template_folder_paths(app):
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
-        'APP_DIRS': True,  # TODO
+        #'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
+        'DIRS': [os.path.join(PROJECT_ROOT, 'apps/shared/templates')],
```

Comment:
Sounds good! I think that as long as we create issues for these changes or
label up the area as a TODO in the code, we should be able to merge in this
pull request.